### PR TITLE
MYB 70

### PR DESCRIPTION
**Pull request by ldimitroff**

### CODE REVIEW

**Comment by ldimitroff** [comment]
Bug: [https://devspark-com.atlassian.net/browse/MYB-90](url) has been created.

File: app/src/main/java/com/mybus/fragment/BusRouteFragment.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        mStopOrigin.setText(mBusRouteResult.getBusRoutes().get(0).getStartBusStopStreetName() + " " + mBusRouteResult.getBusRoutes().get(0).getStartBusStopStreetNumber());
+        mWalkOrigin.setText("Distancia desde el origen: " + WalkDistanceHelper.getDistanceInBlocks(mBusRouteResult.getBusRoutes().get(0).getStartBusStopDistanceToOrigin()));
+
+        if (mBusRouteResult.getType() == 0) {
+            mStopDestination.setText(mBusRouteResult.getBusRoutes().get(0).getDestinationBusStopStreetName() + " " + mBusRouteResult.getBusRoutes().get(0).getDestinationBusStopNumber());
+            mWalkDestination.setText("Distancia hasta el destino: " + WalkDistanceHelper.getDistanceInBlocks(mBusRouteResult.getBusRoutes().get(0).getDestinationBusStopDistanceToDestination()));
```
Comment (lucasdelio) [inline app/src/main/java/com/mybus/fragment/BusRouteFragment.java]: Move hard coded Strings to resources

File: app/src/main/java/com/mybus/MainActivity.java [hunk 2]
```diff
@@ -374,12 +422,47 @@ private void performSearch() {
...
+    }
+
+    /**
+     * Populates the bottom sheet with the Routes found
+     *
+     * @param results
+     */
+    private void populateBottomSheet(List<BusRouteResult> results) {
+        mViewPagerAdapter = new ViewPagerAdapter(getSupportFragmentManager(), MainActivity.this.getLayoutInflater());
+        for (BusRouteResult route : results) {
```
Comment (lucasdelio) [inline app/src/main/java/com/mybus/MainActivity.java:439]: Add check for null results

File: app/src/main/java/com/mybus/MainActivity.java [hunk 3]
```diff
@@ -374,12 +422,47 @@ private void performSearch() {
 
     @Override
     public void onRouteFound(List<BusRouteResult> results) {
-        Toast.makeText(this, "results found", Toast.LENGTH_LONG).show();
-        int i = 0;
-        if(results!=null){
-            for (BusRouteResult route : results) {
-                Log.d(TAG, "result "+i+": "+route.toString());
-                i++;
+        if (results != null && !results.isEmpty()) {
```
Comment (lucasdelio) [inline app/src/main/java/com/mybus/MainActivity.java]: Move null check to populateBottomSheet

File: app/src/main/java/com/mybus/MainActivity.java [hunk 4]
```diff
@@ -374,12 +422,47 @@ private void performSearch() {
...
+            Toast.makeText(this, "results found", Toast.LENGTH_LONG).show();
```
Comment (lucasdelio) [inline app/src/main/java/com/mybus/MainActivity.java]: Remove this toast

File: app/src/main/java/com/mybus/MainActivity.java [hunk 5]
```diff
@@ -374,12 +422,47 @@ private void performSearch() {
...
+
+    /**
+     * Show or hide the bottom sheet Layout
+     *
+     * @param show
+     */
+    private void showBottomSheetResults(boolean show) {
+        if (mBottomSheetBehavior != null) {
+            if (show) {
+                mBottomSheetBehavior.setPeekHeight(100);
```
Comment (lucasdelio) [inline app/src/main/java/com/mybus/MainActivity.java]: Move 100 to resources

**Pull request merged**